cmake/ScriptFiles.cmake

```diff
@@ -59,9 +59,14 @@ set(SOURCE_FILES
     policy_api.sql
     policy_internal.sql
     cagg_utils.sql
-	  telemetry.sql
     bgw_startup.sql)
 
+if(USE_TELEMETRY)
+  list(APPEND SOURCE_FILES with_telemetry.sql)
```

Comment:
Shouldn't `with_telemetry.sql` be before `bgw_startup.sql`? Ideally right after `bgw_scheduler.sql`?

Reply:
It does not really make a difference, but I added it after the telemetry files for good measure.